code/__DEFINES/dcs/signals.dm

```diff
@@ -1097,3 +1097,6 @@
 #define COMSIG_AQUARIUM_BEFORE_INSERT_CHECK "aquarium_about_to_be_inserted"
 #define COMSIG_AQUARIUM_SURFACE_CHANGED "aquarium_surface_changed"
 #define COMSIG_AQUARIUM_FLUID_CHANGED "aquarium_fluid_changed"
+
+//Singularity component signals
+#define COMSIG_SINGULO_SIZE_CHANGE "singulos_size_changed"
```

Comment:
No. We're pretty strict about not having signals that are essentially just procs. If there is nothing other than the singularity itself that hooks to this, then it should not be a signal. This exact conversation was had on the PR itself.

Reply:
@Mothblocks I dont see that conversation when looking there? I only see you being told to do actually use signals instead like I did here by spooky: https://github.com/tgstation/tgstation/pull/55096#discussion_r529380267